src/app/public/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -268,8 +268,9 @@ describe('Dropdown component', function () {
     expect(menu.scrollHeight > menu.clientHeight).toEqual(true);
   }));
 
-  it('should emit when a menu item is selected', fakeAsync(() => {
+  it('should emit when a menu item is clicked', fakeAsync(() => {
     const menuChangesSpy = spyOn(fixture.componentInstance, 'onMenuChanges').and.callThrough();
+    const itemClickSpy = spyOn(fixture.componentInstance, 'onItemClick').and.callThrough();
```

Comment:
This will spy on the consumer-provided `click` event handler, to make sure those come through.